scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala

```diff
@@ -175,7 +175,13 @@ class HtmlRenderer(rootPackage: Member, members: Map[DRI, Member])(using ctx: Do
     def icon(link: SocialLinks) = link.className
     args.socialLinks.map { link =>
       a(href := link.url) (
-        button(cls := s"icon-button ${icon(link)}")
+        if icon(link) == "custom" then
+          Seq(
+            img(cls := s"icon-button ${icon(link)}", src := s"../../../../images/${link.whiteIcon}"),
```

Comment:
I think that the resources should be handled in a better way. This relative link was used in the css, since it was resolved as a url from the css file path. Right now it won't be possible to fetch the images if the scaladoc is hosted from a place other than the root folder of the generated scaladoc. 

Though not an expert on that, so maybe @Florian3k might have some insights.

Reply:
One solution would be to fix the name of the custom logo and to use it in the css like this:

.icon-button.custom::after {
  content: url("../../../../images/icon-buttons/custom/light/default.svg");
}

But when if someone doesn't use a custom, there is an error.